### PR TITLE
Wherein Fethas proves to be evil incarnate

### PR DESCRIPTION
**Pull request by Aurorablade**
Adds Lycancoughy: A virus that makes you cough up pugs, corgi puppys and foxes.

Advanced Pierrorts Throat: Its kinda like the other one...but with a clown mask.

KINGSTONS: I could not remeber the old virus well..but this will hopefully surffice.

Taking feedback on message changes and how to make this better/worse.

:cl: Fethas
rscadd: Viruses Lycancoughy, Adv. Pierrots throat and Kingstons added.
/:cl:

### CODE REVIEW

**Comment by MarcellusPye** [comment]
GET IT MERGED!

**Comment by SamHPurp** [comment]
The highest of RP.

I don't think people will like their snowflakes changing races.

File: code/datums/diseases/kingstons.dm [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+	name = "Kingstons"
+	max_stages = 4
+	spread_text = "Airborne"
+	cure_text = "Milk."
+	cures = list("milk")
+	cure_chance = 75
+	agent = "NYA Virus"
+	viable_mobtypes = list(/mob/living/carbon/human)
+	permeability_mod = 0.75
+	desc = "If left untreated the subject will turn into a felonid. In felines it has..OTHER..effects."
```
Comment (SamHPurp) [inline code/datums/diseases/kingstons.dm]: `felonid` -> `feline`

`In felines it has..OTHER..effects.` -> `In felines it has... OTHER... effects.`

I don't like it, but if you're going to ellipses, use two of them, and you need spaces...

File: code/datums/diseases/kingstons.dm [hunk 2]
```diff
@@ -0,0 +1,46 @@
+/datum/disease/kingstons
+	name = "Kingstons"
+	max_stages = 4
+	spread_text = "Airborne"
+	cure_text = "Milk."
```
Comment (SamHPurp) [inline code/datums/diseases/kingstons.dm]: Why does this one have a period/fullstop and the other does? Determine which is needed, and correct the incorrect one.

File: code/datums/diseases/kingstons.dm [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+	cures = list("milk")
+	cure_chance = 75
+	agent = "NYA Virus"
```
Comment (SamHPurp) [inline code/datums/diseases/kingstons.dm]: cringe

File: code/datums/diseases/kingstons.dm [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+		if(2)
+			if(prob(10))
+				if(affected_mob.get_species() == "Tajaran")
+					to_chat(affected_mob, "<span class='danger'>Something in your throat itches.</span>")
+				else
+					to_chat(affected_mob, "<span class='danger'>You NEED to find a mouse.</span>")
+		if(3)
+			if(prob(10))
+				if(affected_mob.get_species() == "Tajaran")
+					to_chat(affected_mob, "<span class='danger'>You feel something in your throat!.</span>")
```
Comment (SamHPurp) [inline code/datums/diseases/kingstons.dm]: Remove the superfluous period/fullstop.

File: code/datums/diseases/kingstons.dm [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+				else
+					affected_mob.say( pick( list("Mew", "meow", "Nya!") ) )
+		if(4)
+			if(prob(5))
+				if(affected_mob.get_species() == "Tajaran")
+					affected_mob.visible_message("<span class='danger'>[affected_mob] coughs up a hairball!</span>", \
+													"<span class='userdanger'>You cough up a hairball!</span>")
+					affected_mob.Stun(5)
+				else
+					affected_mob.visible_message("<span class='danger'>[affected_mob] form contorts into something more feline!!</span>", \
```
Comment (SamHPurp) [inline code/datums/diseases/kingstons.dm]: Single exclamation

File: code/datums/diseases/lycancoughy.dm [hunk 6]
```diff
@@ -0,0 +1,45 @@
+/datum/disease/lycan
+	name = "Lycancoughy"
+	form = "Infection"
+	max_stages = 4
+	spread_text = "On contact"
+	spread_flags = CONTACT_GENERAL
+	cure_text = "Ethanol"
+	cures = list("ethanol")
+	agent = "Excess Snuggles"
```
Comment (SamHPurp) [inline code/datums/diseases/lycancoughy.dm]: cringe

File: code/datums/diseases/lycancoughy.dm [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+	viable_mobtypes = list(/mob/living/carbon/human, /mob/living/carbon/human/monkey)
+	desc = "If left untreated subject will regurgitate...puppys?"
```
Comment (SamHPurp) [inline code/datums/diseases/lycancoughy.dm]: space after your ellipses

File: code/datums/diseases/lycancoughy.dm [hunk 8]
```diff
@@ -0,0 +1,45 @@
...
+	severity = MEDIUM
+
+/datum/disease/lycan/stage_act()
+	..()
+	switch(stage)
+		if(2) //also changes say, see say.dm
```
Comment (SamHPurp) [inline code/datums/diseases/lycancoughy.dm]: Intentionally nothing at stage 1?

File: code/datums/diseases/lycancoughy.dm [hunk 9]
```diff
@@ -0,0 +1,45 @@
...
+			if(prob(2))
+				to_chat(affected_mob, "<span class='danger'>Your stomach growls!</span>")
+		if(4)
+			if(prob(10))
+				to_chat(affected_mob, "<span class='danger'>Your stomach barks?!</span>")
+			if(prob(5))
+				affected_mob.visible_message("<span class='danger'>[affected_mob] howls!</span>", \
+												"<span class='userdanger'>You howl!</span>")
+				affected_mob.AdjustConfused(rand(6,8))
+				playsound(affected_mob, 'sound/goonstation/voice/howl.ogg', 60, 0, 10)
```
Comment (SamHPurp) [inline code/datums/diseases/lycancoughy.dm]: I hate this audio clip so much, increasing its use seems bad.

File: code/datums/diseases/lycancoughy.dm [hunk 10]
```diff
@@ -0,0 +1,45 @@
...
+			if(prob(3))
+				var/list/puppytype = list(/mob/living/simple_animal/pet/corgi/puppy, /mob/living/simple_animal/pet/pug, /mob/living/simple_animal/pet/fox)
+				var/mob/living/puppypicked = pick(puppytype)
+				affected_mob.visible_message("<span class='danger'>[affected_mob] coughs up [initial(puppypicked.name)]!</span>", \
+													"<span class='userdanger'>You cough up [initial(puppypicked.name)]!?</span>")
```
Comment (SamHPurp) [inline code/datums/diseases/lycancoughy.dm]: remove the superfluous question mark

File: code/datums/diseases/pierrot_throat.dm [hunk 11]
```diff
@@ -26,3 +26,39 @@
...
+	name = "Advanced Pierrot's Throat"
+	max_stages = 4
+	spread_text = "Airborne"
+	cure_text = "Banana products, especially banana bread."
+	cures = list("banana")
+	cure_chance = 75
+	agent = "H0NI<42.B4n4 Virus"
+	viable_mobtypes = list(/mob/living/carbon/human)
+	permeability_mod = 0.75
+	desc = "If left untreated the subject will probably drive others to insanity and go insane themselves."
```
Comment (SamHPurp) [inline code/datums/diseases/pierrot_throat.dm]: cringe

File: code/datums/diseases/pierrot_throat.dm [hunk 12]
```diff
@@ -26,3 +26,39 @@
...
+	..()
+	switch(stage)
+		if(1)
+			if(prob(10))
+				to_chat(affected_mob, "<span class='danger'>You feel a little silly.</span>")
+			if(prob(5))
+				to_chat(affected_mob, "<span class='danger'>You feel like making a joke.</span>")
+		if(2)
+			if(prob(10))
+				to_chat(affected_mob, "<span class='danger'>You don't just start seeing rainbows..YOU ARE RAINBOWS!</span>")
```
Comment (SamHPurp) [inline code/datums/diseases/pierrot_throat.dm]: Cringe.

Also, three dots in ellipses, and again you need a space after them.

**Comment by HugoLuman** [comment]
Eh, it's all too much of a meme. We don't have "TAJARANS ARE VALID HERE," so it's not even a good meme. [s]And makes it feel a bit like wasted effort to write all that lore[/s]

A better meme would be turning folks into vulps (though still probably not a good addition). Or turning other races into _humans_

**Comment by Aurorablade** [comment]
Well I am for locking it down to admin only and making other racial varients if given inspiration

**Comment by Birdtalon** [comment]
Local admin constantly adding admin only things. 👎 👎 👎 👎 👎 👎 👎 👎

**Comment by uraniummeltdown** [comment]
1/2👍 Hyper-realism, "MRP = super seriousness and no fun" are boring
How are you supposed to get these viruses though? Would a non-antag get banned for giving people the virus as a prank?
Advanced Honk is kind of useful but we already have regular Honk
Lycancoughy is fine, doesn't really do anything but free dog mobs
Kingstons is great, funny and inventive. I hope antags who use it don't get banned for 'griefing' or 'ruining someone's round' because someone's OC became a Tajaran

File: code/datums/diseases/lycancoughy.dm (hunk 8 above)
Comment (Aurorablade) [reply code/datums/diseases/lycancoughy.dm]: i am basing it on Severe Anixity here.

**Comment by Aurorablade** [comment]
alot of viruses are locked down already mainly to prevent salt (or it has issues)
i need to consult with someone on how locked down it is.

Its also why i want feedback to try to improve messages or what effects could be added where, again i could add other racial-transformation viruses...Could actually have vox pox.

**Comment by Tayyyyyyy** [comment]
> Hyper-realism, "MRP = super seriousness and no fun" are boring

Strawperson?! When did Sam say he was against fun? It's not hyper realism to not want to have a disease that causes you to randomly cough up dogs. Just because someone has a different, completely reasonable interpretation of MRP from you doesn't mean you should degrade it as "no fun allowed".

**Comment by FalseIncarnate** [comment]
Just for the record: we already do have a virus symptom that makes you cough up butterfly mobs.

As for the argument against adding it as an admin-only thing, I wouldn't use that as a sole reason to oppose things. A lot of times things are added as admin-only because either it is literally meant for admins only (tools and such), we know it will be abused if publically available, or because we want to be able to test the impact better before making the call on making it publically available.

Plus Kingstons is really more a feature return than a new feature. Semantics but whatever.

**Comment by uraniummeltdown** [comment]
@Tayyyyyyy It's not a dig at Sam, I just dislike the trend of removing/preventing silly things for the sole reason of "MRP"

**Comment by pinatacolada** [comment]
hey how about making kingstons behave the same as the slime mutation disease (that turns you into a slime person, then into an actual slime if left untreated), and make it turn you into a taj, then into a cat simple mob

it's even got the added bonus of being consistent with other diseases' behaviour trust me i have a degree

**Comment by uraniummeltdown** [comment]
@pinatacolada Ehh simple mobs are kinda shitty, you can hardly do anything as them and there's no way to become a human again

**Comment by pinatacolada** [comment]
just like with slimes

**Comment by uraniummeltdown** [comment]
Yeah the difference is one is a virus that spreads and is infinite and the other requires work to get black slimes and results in 1 unit of toxin

**Comment by pinatacolada** [comment]
actually just remembered that jungle fever is kinda similar to this too, but its revertible unless you were lucky enough to be a species without a monkey form, but yea

still considering this is admin only i think its k

**Comment by Aurorablade** [comment]
please note for people with gripes of messages 
"Taking feedback on message changes and how to make this better/worse."

**Comment by Aurorablade** [comment]
and i will be coding in stuff from comments..when my brain recovers from work.... *screeches and runs to box*

**Comment by adamkad1** [comment]
diease to turn into tajarans? now you make me want someone to make vox pox

**Comment by Aurorablade** [comment]
wellllllllllllll... *looks at some code she is working on*

File: code/datums/diseases/kingstons.dm [hunk 13]
```diff
@@ -40,7 +40,58 @@
...
+	switch(stage)
+		if(1)
+			if(prob(10))
+				to_chat(affected_mob, "<span class='notice'>You feel awkward.</span>")
+		if(2)
+			if(prob(10))
+				to_chat(affected_mob, "<span class='danger'>You itch.</span>")
+		if(3)
+			if(prob(10))
+				to_chat(affected_mob, "<span class='danger'>Your skin starts to flake!</span>")
```
Comment (SamHPurp) [inline code/datums/diseases/kingstons.dm]: `flake` -> `snowflake` 🤣

File: code/datums/diseases/kingstons.dm [hunk 14]
```diff
@@ -0,0 +1,97 @@
...
+	max_stages = 4
+	spread_text = "Airborne"
+	cure_text = "Milk"
+	cures = list("milk")
+	cure_chance = 50
+	agent = "Nya Virus"
+	viable_mobtypes = list(/mob/living/carbon/human)
+	permeability_mod = 0.75
+	desc = "If left untreated the subject will turn into a feline. In felines it has... OTHER... effects."
+	severity = MEDIUM
```
Comment (Citinited) [inline code/datums/diseases/kingstons.dm]: Should be more severe than `MEDIUM`, given it turns you into a cat

File: code/datums/diseases/kingstons.dm [hunk 15]
```diff
@@ -0,0 +1,97 @@
...
+		if(4)
+			if(prob(5))
+				if(affected_mob.get_species() == "Tajaran")
+					affected_mob.visible_message("<span class='danger'>[affected_mob] coughs up a hairball!</span>", \
+													"<span class='userdanger'>You cough up a hairball!</span>")
+					affected_mob.Stun(5)
+				else
+					affected_mob.visible_message("<span class='danger'>[affected_mob] form contorts into something more feline!</span>", \
+													"<span class='userdanger'>YOU TURN INTO A TAJARAN!</span>")
+					if(!affected_mob.get_species() != "Tajaran")
```
Comment (Citinited) [inline code/datums/diseases/kingstons.dm]: This line is confusing... I'd just remove it entirely given you already checked `affected_mob` isn't a tajaran

File: code/datums/diseases/kingstons.dm (hunk 13 above)
Comment (Aurorablade) [reply code/datums/diseases/kingstons.dm]: that wasn't even my intent but now that you have that thought in my head...

File: code/datums/diseases/kingstons.dm [hunk 16]
```diff
@@ -0,0 +1,97 @@
...
+	name = "Advanced Kingstons Syndrome"
+	max_stages = 4
+	spread_text = "Airborne"
+	cure_text = "Plasma"
+	cures = list("plasma")
+	cure_chance = 50
+	agent = "AMB45DR Bacteria"
+	viable_mobtypes = list(/mob/living/carbon/human)
+	permeability_mod = 0.75
+	desc = "If left untreated the subject will mutate to a diffrent species."
```
Comment (Citinited) [inline code/datums/diseases/kingstons.dm]: `different` not `diffrent`

File: code/datums/diseases/kingstons.dm [hunk 17]
```diff
@@ -0,0 +1,97 @@
...
+	severity = MEDIUM
+	var/list/virspecies = list("Human", "Tajaran", "Unathi", "Skrell", "Grey")//no karma races sorrys.
+	var/list/virsuffix = list("pox", "rot", "flu", "Cough", "-gitis", "cold", "Rash")
```
Comment (Citinited) [inline code/datums/diseases/kingstons.dm]: `Cough` and `Rash` should be decapitalised

File: code/datums/diseases/kingstons.dm [hunk 18]
```diff
@@ -0,0 +1,97 @@
...
+	max_stages = 4
+	spread_text = "Airborne"
+	cure_text = "Plasma"
+	cures = list("plasma")
+	cure_chance = 50
+	agent = "AMB45DR Bacteria"
+	viable_mobtypes = list(/mob/living/carbon/human)
+	permeability_mod = 0.75
+	desc = "If left untreated the subject will mutate to a diffrent species."
+	severity = MEDIUM
```
Comment (Citinited) [inline code/datums/diseases/kingstons.dm]: See above re: severity

File: code/datums/diseases/lycancoughy.dm [hunk 19]
```diff
@@ -0,0 +1,44 @@
...
+	name = "Lycancoughy"
+	form = "Infection"
+	max_stages = 4
+	spread_text = "On contact"
+	spread_flags = CONTACT_GENERAL
+	cure_text = "Ethanol"
+	cures = list("ethanol")
+	agent = "Excess Snuggles"
+	viable_mobtypes = list(/mob/living/carbon/human, /mob/living/carbon/human/monkey)
+	desc = "If left untreated subject will regurgitate... puppys?"
```
Comment (Citinited) [inline code/datums/diseases/lycancoughy.dm]: I'd reword this (and make it not a question), given it's meant to be a description in a medical database.

File: code/datums/diseases/lycancoughy.dm [hunk 20]
```diff
@@ -0,0 +1,44 @@
...
+				to_chat(affected_mob, "<span class='danger'>Your stomach growls!</span>")
+		if(4)
+			if(prob(10))
+				to_chat(affected_mob, "<span class='danger'>Your stomach barks?!</span>")
+			if(prob(5))
+				affected_mob.visible_message("<span class='danger'>[affected_mob] howls!</span>", \
+												"<span class='userdanger'>You howl!</span>")
+				affected_mob.AdjustConfused(rand(6,8))
+			if(prob(3))
+				var/list/puppytype = list(/mob/living/simple_animal/pet/corgi/puppy, /mob/living/simple_animal/pet/pug, /mob/living/simple_animal/pet/fox)
```
Comment (Citinited) [inline code/datums/diseases/lycancoughy.dm]: They're coughing up puppies - give them brute damage from it, the human throat isn't big enough to accomodate this

File: code/datums/diseases/pierrot_throat.dm [hunk 21]
```diff
@@ -26,3 +26,39 @@
 		if(4)
 			if(prob(5))
 				affected_mob.say( pick( list("HONK!", "Honk!", "Honk.", "Honk?", "Honk!!", "Honk?!", "Honk...") ) )
+
+
+/datum/disease/pierrot_throat/advanced
+	name = "Advanced Pierrot's Throat"
+	max_stages = 4
```
Comment (Citinited) [inline code/datums/diseases/pierrot_throat.dm]: Most of these vars are already defined under `/datum/disease/pierrot_throat` and as such don't need to be redefined here

File: code/datums/diseases/pierrot_throat.dm [hunk 22]
```diff
@@ -26,3 +26,39 @@
...
+	permeability_mod = 0.75
+	desc = "If left untreated the subject will probably drive others to insanity and go insane themselves."
+	severity = DANGEROUS
+
+/datum/disease/pierrot_throat/advanced/stage_act()
+	..()
+	switch(stage)
+		if(1)
+			if(prob(10))
+				to_chat(affected_mob, "<span class='danger'>You feel a very silly.</span>")
```
Comment (Citinited) [inline code/datums/diseases/pierrot_throat.dm]: Either `You feel a little silly` or `You feel very silly`

File: code/datums/diseases/kingstons.dm (hunk 13 above)
Comment (KasparoVy) [reply code/datums/diseases/kingstons.dm]: your skin starts to :snowflake:

File: code/datums/diseases/lycancoughy.dm (hunk 19 above)
Comment (KasparoVy) [reply code/datums/diseases/lycancoughy.dm]: puppies*

File: code/datums/diseases/pierrot_throat.dm (hunk 22 above)
Comment (KasparoVy) [reply code/datums/diseases/pierrot_throat.dm]: you become the silliness

File: code/datums/diseases/lycancoughy.dm (hunk 20 above)
Comment (taukausanake) [reply code/datums/diseases/lycancoughy.dm]: Not unless we're turning into VORE. Related question: is it still vore if it's coming _out_ of someone's mouth?

File: code/datums/diseases/pierrot_throat.dm (hunk 22 above)
Comment (taukausanake) [reply code/datums/diseases/pierrot_throat.dm]: You feel a little very silly

File: code/datums/diseases/lycancoughy.dm (hunk 20 above)
Comment (SamHPurp) [reply code/datums/diseases/lycancoughy.dm]: @taukausanake I had the same thoughts but decided against voicing them...

Comment (taukausanake) [reply code/datums/diseases/lycancoughy.dm]: Don't worry, leave it to me to ask the tough questions

Comment (Citinited) [reply code/datums/diseases/lycancoughy.dm]: It's a stretch (snicker) to say that this is vore at all.

A puppy travelling from your stomach to your mouth (through your throat) *will* fuck you up internally and as such should have the relevant consequences

**Comment by Denthamos** [comment]
This feels like it will just cause half of the playerbase to cryo as soon as they catch the tarjan one.  We already get a lot of people cryoing when changlings Tsting half of the server into a specific meme-person.

And sure you can add it in as admin only but that just adds to the things which will be forgotten in the code.  Why add things that will only get used maybe a a few times at most and then be forgotten?  I mean we still have RIG suits in the code that have so many issues with them, and have been in the code forever.

File: code/datums/diseases/pierrot_throat.dm (hunk 21 above)
Comment (Aurorablade) [reply code/datums/diseases/pierrot_throat.dm]: I would rather keep it just incase the values need to be tweaked.

File: code/datums/diseases/pierrot_throat.dm (hunk 22 above)
Comment (Aurorablade) [reply code/datums/diseases/pierrot_throat.dm]: little very kinda silly. Its a clown virus....

**Comment by Aurorablade** [comment]
most of the issues with rigs are fixed its just a matter of balance and implementation.

File: code/datums/diseases/kingstons.dm [hunk 23]
```diff
@@ -0,0 +1,96 @@
...
+				else
+					affected_mob.say( pick( list("Mew", "meow", "Nya!") ) )
+		if(4)
+			if(prob(5))
+				if(affected_mob.get_species() == "Tajaran")
+					affected_mob.visible_message("<span class='danger'>[affected_mob] coughs up a hairball!</span>", \
+													"<span class='userdanger'>You cough up a hairball!</span>")
+					affected_mob.Stun(5)
+				else
+					affected_mob.visible_message("<span class='danger'>[affected_mob] form contorts into something more feline!</span>", \
```
Comment (imsxz) [inline code/datums/diseases/kingstons.dm]: [affected_mob]'s

File: code/datums/diseases/pierrot_throat.dm (hunk 21 above)
Comment (Citinited) [reply code/datums/diseases/pierrot_throat.dm]: In that case you can just redefine them if you need to - no need to have the redundancy

**Comment by Citinited** [comment]
@Denthamos Paradise used to have Kingstons and I don't recall masses of people cryoing over it.

**Comment by Dumbdumn5** [comment]
@Denthamos a few people even used Kingstons in a form of persistence to make their character development a tad more interesting and change things up a bit (Hawke Boetirson being the first to come to mind.) Also, want to keep in mind that not everything has to be Low or High RP to be Med RP, and a bit of nonseriousness into the mix would be fun. I'd enjoy it more if it were just added as very rare viruses so that we don't have clowns, puppies, or kingstons every other shift, as it might get dull.

**Comment by Kyep** [comment]
@Allfd

**Comment by Allfd** [comment]
Yeah, I agree with @Citinited  This was a thing before Viruses were refactored and we survived just fine.

Right now admins can already emulate this though with the transformation virus.  It would make sense for some of these to be from virology.  Its been a very long time since I heard anyone curse virology over the radio.

**Comment by Aurorablade** [comment]
LYNCH THE VIRO

File: code/datums/diseases/kingstons.dm [hunk 24]
```diff
@@ -0,0 +1,95 @@
...
+					to_chat(affected_mob, "<span class='danger'>Something in your throat itches.</span>")
+				else
+					to_chat(affected_mob, "<span class='danger'>You NEED to find a mouse.</span>")
+		if(3)
+			if(prob(10))
+				if(affected_mob.get_species() == "Tajaran")
+					to_chat(affected_mob, "<span class='danger'>You feel something in your throat!</span>")
+					affected_mob.emote("cough")
+				else
+					affected_mob.say( pick( list("Mew", "meow", "Nya!") ) )
```
Comment (tigercat2000) [inline code/datums/diseases/kingstons.dm]: Too many spaces, `affected_mob.say(pick(list("Mew", "Meow!", "Nya!~")))`

File: code/datums/diseases/kingstons.dm (hunk 23 above)
Comment (tigercat2000) [reply code/datums/diseases/kingstons.dm]: ^ what he said

File: code/datums/diseases/lycancoughy.dm [hunk 25]
```diff
@@ -0,0 +1,45 @@
...
+				affected_mob.emote("cough")
+			if(prob(2))
+				to_chat(affected_mob, "<span class='danger'>Your stomach growls!</span>")
+		if(4)
+			if(prob(10))
+				to_chat(affected_mob, "<span class='danger'>Your stomach barks?!</span>")
+			if(prob(5))
+				affected_mob.visible_message("<span class='danger'>[affected_mob] howls!</span>", \
+												"<span class='userdanger'>You howl!</span>")
+				affected_mob.AdjustConfused(rand(6,8))
```
Comment (tigercat2000) [inline code/datums/diseases/lycancoughy.dm]: Spacing after commas.

**Comment by DarkPyrolord** [comment]
👍

**Comment by tigercat2000** [comment]
Yess bring back kingstons
